--- neard-0.15/Makefile.am.orig	2014-12-08 11:02:07.000000000 +0100
+++ neard-0.15/Makefile.am	2014-12-20 13:32:14.336233139 +0100
@@ -10,11 +10,11 @@
 			include/tlv.h include/setting.h include/device.h \
 			include/nfc_copy.h include/snep.h
 
-nodist_include_HEADERS = include/version.h
+nodist_pkginclude_HEADERS = include/version.h
 
 noinst_HEADERS = include/dbus.h
 
-local_headers = $(foreach file,$(pkginclude_HEADERS) $(nodist_include_HEADERS) \
+local_headers = $(foreach file,$(pkginclude_HEADERS) $(nodist_pkginclude_HEADERS) \
 			$(noinst_HEADERS), include/near/$(notdir $(file)))
 
 gdbus_sources = gdbus/gdbus.h gdbus/mainloop.c gdbus/watch.c \
